reconciletags/BUILD

```diff
@@ -0,0 +1,41 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@subpar//:subpar.bzl", "par_binary")
+
+load(
+    "@io_bazel_rules_go//go:def.bzl",
+    "go_prefix",
+)
+
+load(
+    "@io_bazel_rules_docker//docker:docker.bzl",
+    "docker_push",
+    "docker_build",
+)
+
+py_binary(
```

Comment:
Can you add a "par_binary" target here as well, since you're loading in subpar/par_binary on line 3?